projects/generate_confs/parser.ypp

```diff
@@ -278,6 +278,8 @@
 //ellesettete
 %type <ellesettete_meas> ellesettete_meas
 %token TK_MEAS_ELLESETTETE
+%type <int_numb> method_flag
+%token TK_METHOD_FLAG
```

Comment:
you can add `TK_NUMERICAL` and `TK_ANALYTICAL` matching `Numerical` and ```Analytical``` strings to write more informative input file (the matching is in the lex file)